ehr-garden/openemr/Dockerfile

```
@@ -67,22 +69,26 @@ COPY upgrade/docker-version \
      upgrade/fsupgrade-4.sh \
      upgrade/fsupgrade-5.sh \
      upgrade/fsupgrade-6.sh \
-     /root/
+     /root
```

I'm fairly sure that a slash at the end of the destination is required when `COPY`ing multiple sources
```suggestion
     /root/
```

---

Good catch, that was an unintentional mistake on my part. Thanks!